storage/storage_tests/storage_tests.cpp

```diff
@@ -493,11 +500,13 @@ UNIT_TEST(StorageTest_DownloadMapAndRoutingSeparately)
     }
   });
 
-  TIndex const index = storage.FindIndexByFile("Azerbaijan");
+  bool const isSingleMwm = version::IsSingleMwm(storage.GetCurrentDataVersion());
```

Comment:
The whole test makes sense only when we have separate map and routing file. I'd propose to early exit if !version::IsSingleMwm.